src/scheduler/fifo.c

```diff
@@ -2327,3 +2350,358 @@ next_job(status *policy, server_info *sinfo, int flag)
 	}
 	return rjob;
 }
+
+/**
+ * @brief
+ *	Helper function used to copy the given source string to the destination string. It also
+ *	frees the destination and then allocates required memory before copying.
+ *
+ * @param[in] dest - Address of pointer to destination string
+ * @param[in] dest - pointer to source string
+ *
+ * @retval
+ * @return 0 - Failure
+ * @return  1 - Success
+ *
+ * @par Side Effects:
+ *	None
+ *
+ *
+ */
+static int
+copy_attr_value(char **dest, char *src)
```

Comment:
you can either make it more generic and call it pbs_strcpy and put it in misc_utils.c or just make use of pbs_strcat to the same thing (just reset the destination to '\0' everytime before copy.